### DISCUSSION TITLE
test cases for pseudo-binary decoding

### DISCUSSION THREAD

**Discussion by ghost** [opendcs/opendcs#282] [Q&A]
Does anyone know of available data to test pseudo-binary decoding?

**Comment by MikeNeilson** [comment]
Not off the top of my head, but the dbexport.log on this issue https://github.com/opendcs/opendcs/issues/255 likely contains a few psuedo binary stations.

The various vendors (Like Sutron, FTS, Microcom, etc) should have examples in their respective manuals.

**Reply by MikeNeilson** [reply]
There is this portion of the manual describing some psuedo binary data, https://opendcs-env.readthedocs.io/en/latest/decoding-guide.html#pseudo-binary-and-pure-binary-fields, but it clearly needs some actual examples.

**Reply by ghost** [reply]
That's what I followed. But it'd be good to test with some actual data
